inc/template-tags.php

```diff
@@ -17,9 +17,6 @@
 	 */
 	function twenty_twenty_one_posted_on() {
 		$time_string = '<time class="entry-date published updated" datetime="%1$s">%2$s</time>';
-		if ( get_the_time( 'U' ) !== get_the_modified_time( 'U' ) ) {
-			$time_string = '<time class="entry-date published" datetime="%1$s">%2$s</time> <time class="updated" datetime="%3$s">%4$s</time>';
-		}
 
 		$time_string = sprintf(
```

Comment:
Since the second half of `$time_string` is not used anymore, you don't need to pass the last two args here (`get_the_modified_date`) - there will always only be two placeholders.